# Copyright 2021, Pick-It NV.
# All rights reserved.

uint8 METHOD_LOAD_BY_CONFIG_NAME = 0
uint8 METHOD_LOAD_BY_CONFIG_ID   = 1

uint8     method            # Defines if the configuration is laoded by name or by ID.
string    config            # Name of the configuration. Only used when method is METHOD_LOAD_BY_CONFIG_NAME.
uint32    config_id         # ID of the configuration. Only used when method is METHOD_LOAD_BY_CONFIG_ID.
bool      set_persistent    # If true, this configuration will be automatically be loaded when the system boots.
---
bool      success           # indicate successful run of triggered service
string    message           # informational, e.g. for error messages
